Source/WebCore/html/CanvasBase.h

```diff
@@ -109,7 +112,8 @@ class CanvasBase {
 
     GraphicsClient* graphicsClient() const;
 
-    virtual void didDraw(const std::optional<FloatRect>&) = 0;
+    void didDraw(const std::optional<FloatRect>& rect) { return didDraw(rect, ShouldApplyPostProcessingToDirtyRect::Yes); }
+    virtual void didDraw(const std::optional<FloatRect>&, const ShouldApplyPostProcessingToDirtyRect);
```

Comment:
Forgot to mention this before: const of by-value function parameters are not useful, so you could remove the const from ShouldApplyPostProcessingToDirtyRect.